### PR TITLE
Chore/mar 928 fix vulnerability

### PR DESCRIPTION
**Pull request by alber-devo**
Fix SNYK-JS-CRYPTOJS-6028119 vulnerability by updating @devoinc/js-helper transitive dependency

https://security.snyk.io/vuln/SNYK-JS-CRYPTOJS-6028119

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v16.19.1
```
Comment (tripu) [inline .nvmrc:1]: Why do we need this specific version of Node.js?

In any case, better to set it to `lts/gallium`.

If this is now necessary, it should be in line with [property `engines`](https://github.com/DevoInc/browser-sdk/blob/efe464f2b15b3db1e9e919ec8eeac2be8cf52749/package.json#L91-L94).

**Review by tripu — CHANGES_REQUESTED** [review]

File: .nvmrc (hunk 1 above)
Comment (alber-devo) [reply .nvmrc]: ok! just removed this `.nvmrc`  file with the default npm version ✅👍

**Review by tripu — APPROVED** [review]

**Comment by tripu** [comment]
PR blocked because you're not signing your commits with a GPG or SSH key, @alber-devo. I guess you never do that? I like to sign my contributions, and it's easy to do, but if that's not a popular feature, I'll relax the settings in this repo&hellip;

**Comment by alber-devo** [comment]
I see your point @tripu 👀. However, as this is a public repo, I would suggest relaxing this requirements, as it may limit some future contributions from people outside the company who doesn't follow this rule. 

Also, looking at another Devo repositories, such as [App-Developer-Kit](https://github.com/DevoInc/App-Developer-Kit) or [js-helper](https://github.com/DevoInc/js-helper/pull/15), they don't seem to apply this rule either.

**Comment by Worvast** [comment]
I approve this MR because we have a lote of urgency snd web can verify its the real Alberto who make this commits. 

I need revise whey its the signed query active when its not a Devo org policy, and whey its the CLA sign deactivate, when its a mandatory requirement. But for this we need a exception because i back 6 Nov of vacations and we cant wait for that.

**Comment by tripu** [comment]
As I said, I know many (most?) people don't sign their Git commits&nbsp;&mdash;&nbsp;although it's a measure that provides a better degree of security and [it's very easy to adopt](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification). I think we should enforce that, much in the same way we enforce 2FA.

OK for now to not mandate that in this repo.

**Comment by tripu** [comment]
I just disabled that option here.

**Pull request merged**